pkg/webhook/cloneset/validating/validation.go

```diff
@@ -92,6 +92,13 @@ func (h *CloneSetCreateUpdateHandler) validateCloneSetSpec(spec, oldSpec *appsv1
 	allErrs = append(allErrs, h.validateScaleStrategy(&spec.ScaleStrategy, oldScaleStrategy, metadata, fldPath.Child("scaleStrategy"))...)
 	allErrs = append(allErrs, h.validateUpdateStrategy(&spec.UpdateStrategy, int(*spec.Replicas), fldPath.Child("updateStrategy"))...)
 
+	if spec.ProgressDeadlineSeconds != nil {
+		allErrs = append(allErrs, apivalidation.ValidateNonnegativeField(int64(*spec.ProgressDeadlineSeconds), fldPath.Child("progressDeadlineSeconds"))...)
+		if *spec.ProgressDeadlineSeconds <= spec.MinReadySeconds {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("progressDeadlineSeconds"), spec.ProgressDeadlineSeconds, "must be greater than minReadySeconds"))
```

Comment:
I believe this validation may be inappropriate, or the default value should be removed. My reasons are as follows:

1. Before this feature was introduced, MinReadySeconds might already have been set to a value greater than 600. Setting a default of 600 would trigger the error in this check.
2. I also don’t think it’s appropriate to use MinReadySeconds as the default for ProgressDeadlineSeconds.

As a result, Kruise Rollout could hit this error and cause the release to fail after an upgrade.